integrationtest/wildfly/src/test/java/org/hibernate/search/test/integration/jms/transaction/TransactionalJmsMasterSlaveIT.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.search.test.integration.jms.transaction;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.jboss.shrinkwrap.resolver.api.maven.MavenFormatStage;
+import org.jboss.shrinkwrap.resolver.api.maven.MavenStrategyStage;
+import org.junit.runner.RunWith;
+
+import org.hibernate.search.engine.Version;
+import org.hibernate.search.test.integration.VersionTestHelper;
+import org.hibernate.search.test.integration.jms.DeploymentJmsMasterSlave;
+import org.hibernate.search.test.integration.jms.SearchNewEntityJmsMasterSlave;
+import org.hibernate.search.test.integration.jms.util.RegistrationConfiguration;
+
+/**
+ * Execute the tests in {@link SearchNewEntityJmsMasterSlave} adding the the dependencies as jars in the
+ * deployments.
+ *
+ * @author Davide D'Alto
+ * @author Sanne Grinovero
+ */
+@RunWith(Arquillian.class)
+public class TransactionalJmsMasterSlaveIT extends TransactionalJmsMasterSlave {
+
+	private static final File tmpDir = RegistrationConfiguration.createTempDir();
+
+	/**
+	 * Lazy initialization of the libraries since Maven is painfully slow.
+	 */
+	private static class LibrariesLoader {
+		public static final File[] LIBRARIES = init();
+
+		private static File[] init() {
+			final String currentVersion = Version.getVersionString();
```

Comment:
looks like it did the trick and the failure in the IDE is now consistent with the CLI. I've made one of the private method public